### PR TITLE
Adding more options for more scenarios

### PR DESCRIPTION
**Pull request by DonaldKallman**
Looping and delta support added, also made the script runnable from anywhere.

### CODE REVIEW

**Comment by DarthHater** [comment]
@DonaldKallman sorry I never saw this! Still interested in having it merged?!

**Comment by DonaldKallman** [comment]
Yeah, I completely forgot about this too, I think it would be useful to some.

File: mavenimport.sh [hunk 1]
```diff
@@ -3,15 +3,61 @@
...
+  else
+    echo "Uploading to ${NEXUS_URL}/repository/${REPO_DEST}"
+    time find . -type f -not -path './mavenimport\.sh*' -not -path '*/\.*' -not -path '*/\^archetype\-catalog\.xml*' -not -path '*/\^maven\-metadata\-local*\.xml' -not -path '*/\^maven\-metadata\-deployment*\.xml' | sed "s|^\./||" | xargs -I '{}' curl -u "${USERNAME}:${PASSWORD}" -X PUT -v -T {} ${NEXUS_URL}/repository/${REPO_DEST}/{} ;
+  fi
+else
+	cd ${NEXUS_PATH}/${REPO_NAME}
+  if [[ ${REPO_DEST} == "" ]]; then
+    echo "Uploading to ${NEXUS_URL}/repository/${REPO_NAME}"
+	  time find . -type f -not -path './mavenimport\.sh*' -not -path '*/\.*' -not -path '*/\^archetype\-catalog\.xml*' -not -path '*/\^maven\-metadata\-local*\.xml' -not -path '*/\^maven\-metadata\-deployment*\.xml' | sed "s|^\./||" | xargs -I '{}' curl -u "${USERNAME}:${PASSWORD}" -X PUT -v -T {} ${NEXUS_URL}/repository/${REPO_NAME}/{} ;
```
Comment (lghakamo-paf) [inline mavenimport.sh:58]: Looking at the commit history it seems that the `-newermt ${DELTA_DATE}` part went missing.

File: mavenimport.sh [hunk 2]
```diff
@@ -3,15 +3,61 @@
...
+  else
+    echo "Uploading to ${NEXUS_URL}/repository/${REPO_DEST}"
+    time find . -type f -not -path './mavenimport\.sh*' -not -path '*/\.*' -not -path '*/\^archetype\-catalog\.xml*' -not -path '*/\^maven\-metadata\-local*\.xml' -not -path '*/\^maven\-metadata\-deployment*\.xml' | sed "s|^\./||" | xargs -I '{}' curl -u "${USERNAME}:${PASSWORD}" -X PUT -v -T {} ${NEXUS_URL}/repository/${REPO_DEST}/{} ;
```
Comment (lghakamo-paf) [inline mavenimport.sh:61]: Also this is missing the `-newermt ${DELTA_DATE}` part.

**Pull request closed without merging**